transforms/code/repo_level_ordering/ray/README.md

```diff
@@ -0,0 +1,159 @@
+# Repo Level Order Ray Transform 
+
+Please see the set of
+[transform project conventions](../../../README.md#transform-project-conventions)
+for details on general project conventions, transform configuration,
+testing and IDE set up.
+
+## Summary 
+
+This transform requires the input data to have the following columns atleast: 
+
+- repo name: Name of the repo, it is used for grouping in this transform.
+- title : Which is usually file path.
+- language: Programming language of content
+
+The input data for this transform should be in parquet format. The input data is expected to have code data arranged in rows
+such that each row represents a file. The required columns in the input data shoud correspond to a) repository name b) file path
+c) content. This transform supports searching the repo across the whole dataset and writing the files of a repo as a single 
+parquet file. 
+
+The transform gives the option to write the repo to file in the following ways.
+
+a) sort the repo content by file path and write a parquet with multiple rows
```

Comment:
This could use some more description.  It is not clear to me what is going on here.  An example might help.